init.lua

```diff
@@ -366,6 +354,7 @@ local on_digiline_receive_string = function(pos, node, channel, msg)
 				local skip = tonumber(string.sub(msg, 12))
 				led_marquee.scroll_text(pos, nil, skip)
 			elseif msg == "get" then -- get the master panel's displayed char as ASCII numerical value
+				-- luacheck: ignore 631
```

Comment:
what is this ignoring? seems bad practice

Reply:
631	Line is too long
https://luacheck.readthedocs.io/en/stable/warnings.html